src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java

```diff
@@ -258,9 +258,28 @@ private static StringBuilder getAdhocReportCompletedTypeRow(AdhocReportType adho
         return sb;
     }
 
+    private static StringBuilder getLiveCaseLoadReportSummaryHdr(ListingData listingData) {
+        var sb = new StringBuilder();
+        AdhocReportType localReportSummaryHdr = listingData.getLocalReportsSummaryHdr();
+        if (localReportSummaryHdr != null) {
```

Comment:
If the localReportsSummaryHdr was null wouldn't you still want to show the data on the report i.e. it would still show "Multiples: 0 Singles: 0 Total: 0" (or whatever)